src/output.cpp

```diff
@@ -414,16 +415,20 @@ void print_batch_keff()
 
 //==============================================================================
 
-void show_time(const char* label, double value)
+void show_time(const char* label, double secs, int indent_level=0)
 {
-  std::cout << " " << std::setw(33) << std::left << label << " = "
-    << std::setw(10) << std::right << value << " seconds\n";
+  for (int i = 0; i < indent_level; ++i) {
```

Comment:
This loop can be reduced to `std::cout << std::string( 2 * indent_level, ' ');'

Reply:
Nice and succinct. Just updated this; thanks!